### PR TITLE
Mouseflow for the 1%

### PR DESCRIPTION
**Pull request by lc512k**
and `/signup`

cc @commuterjoy @adambraimbridge

### CODE REVIEW

File: tracking/js/third-party/mouseflow.js [hunk 1]
```diff
@@ -6,14 +6,44 @@ const getCookieValue = function (key) {
...
+					case 404:
+						return {};
+					case 200:
+						return response.json();
+					default:
+						throw new Error(`${response.status} - ${response.statusTest}`);
+				}
+			})
+			.then (function (session) {
+				let isNavigationSample = parseInt(session.passportId) % 100 === 0; // 1%
```
Comment (adambraimbridge) [inline tracking/js/third-party/mouseflow.js]: Perhaps `device.spoorId` instead of `session.passportId `?

Comment (lc512k) [inline tracking/js/third-party/mouseflow.js]: `passportId` seemed like a nice, evenly distributed, digit-only string I could quickly `parseInt` & `mod` :) Also @commuterjoy mentioned spoorIds not having an even distribution IIRC?

**Comment by lc512k** [comment]
@commuterjoy any (`parseInt`able, evenly distrd) alternatives to passportId if that's being deprecated?

File: tracking/js/third-party/mouseflow.js [hunk 2]
```diff
@@ -6,14 +6,45 @@ const getCookieValue = function (key) {
...
+					case 404:
+						return {};
+					case 200:
+						return response.json();
+					default:
+						throw new Error(`${response.status} - ${response.statusTest}`);
+				}
+			})
+			.then (function (session) {
+				if (parseInt(session.passportId) % 100 === 0) { // 1%
```
Comment (commuterjoy) [inline tracking/js/third-party/mouseflow.js]: this is deprecated, so perhaps switch over to the uuid at some point?

Comment (lc512k) [inline tracking/js/third-party/mouseflow.js]: @commuterjoy how is uuid generated? Can I safely `mod 100` over the decimal value of the last segment, ie `86976528967924` for `4f1acd1ce8f4` of `c58d7528-d4d5-4cd3-aff8-4f1acd1ce8f4`?

**Pull request merged**